app/eth2wrap/eth2wrap_test.go

```diff
@@ -276,3 +288,53 @@ func TestBlockAttestations(t *testing.T) {
 	require.NoError(t, err)
 	require.Empty(t, resp)
 }
+
+func TestOneDown(t *testing.T) {
+	ctx := context.Background()
+	bmock, err := beaconmock.New()
+	require.NoError(t, err)
+
+	addresses := []string{
+		bmock.Address(),                // Valid
+		"http://222.222.222.222:22222", // Invalid
+	}
+
+	eth2Cl, err := eth2wrap.NewMultiHTTP(ctx, time.Second, addresses...)
+	require.NoError(t, err)
+
+	_, err = eth2Cl.SlotDuration(ctx)
+	require.NoError(t, err)
+}
+
+func TestLazy(t *testing.T) {
```

Comment:
can we also have a testcase in this test where we have 3 addresses with following testcases:
- All disabled.
- One enabled and two disabled
- Two enabled and one disabled

This can replace above `TestOneDown` test

Reply:
not sure this add a lot of value additional value, but you are welcome to add it if you want